app/src/main/res/layout/share_layout.xml

```diff
@@ -29,6 +30,11 @@
             android:src="@drawable/ic_close" />
     </RelativeLayout>
 
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="Users who are shared on this list:"
```

Comment:
shouldn't be a variable on strings.xml?

Reply:
yeah, should be, give me a sec i will fix it

Reply:
wait, it's already fixed, are you on the last commit?